src/OpenLoco/OpenLoco.cpp

```diff
@@ -329,6 +331,21 @@ namespace OpenLoco
         call(0x004BE5EB, regs);
     }
 
+    // 0x004BE65E
+    void exitCleanly()
+    {
+        auto tempFilePath = Environment::getPathNoWarning(Environment::path_id::_1tmp);
```

Comment:
What about the sending out of the quit messages and calling audio_close?

Reply:
As I understand it, `post_quit_message` is setting the return code. I think `exit(0)` covers that.

You're right about `audio_close` being omitted. We haven't implemented that function yet, though, so that needs doing as well.